ex04.ex

```diff
@@ -55,7 +58,9 @@ defmodule Ex04 do
 
   """
 
```

Comment:
### -1

Very nice, except `min` is a library function, and the rubric said not to use them.